cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html

```diff
@@ -70,7 +69,7 @@
             {% endif %}
 
             <img src="{{ photo.url }}"
-                 alt="{{ photo.alt }}"
+                 alt="{{ photo.title }}"
```

Comment:
`title` is a wagtail specific field for images. It renders the title they were uploaded with.